adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/MqttProtocolAdapterProperties.java

```diff
@@ -47,48 +47,9 @@ public MqttProtocolAdapterProperties() {
      */
     public MqttProtocolAdapterProperties(final MqttProtocolAdapterOptions options) {
         super(options.adapterOptions());
-        setCommandAckTimeout(options.commandAckTimeout());
         setSendMessageToDeviceTimeout(options.sendMessageToDeviceTimeout());
     }
 
-    /**
-     * Gets the waiting for acknowledgement timeout in milliseconds for commands published with QoS 1.
-     * <p>
-     * This timeout is used by the MQTT adapter for commands published with QoS 1. If there is no acknowledgement
-     * within this time limit, then the command is settled with the <em>released</em> outcome.
-     * <p>
-     * The default value is {@link #DEFAULT_COMMAND_ACK_TIMEOUT}.
-     *
-     * @deprecated Use {@link #getSendMessageToDeviceTimeout()} instead.
-     *
-     * @return The timeout in milliseconds.
-     */
-    @Deprecated(forRemoval = true)
-    public final int getCommandAckTimeout() {
-        return commandAckTimeout;
```

Comment:
The `commandAckTimeout` field, along with `DEFAULT_COMMAND_ACK_TIMEOUT` and `getEffectiveSendMessageToDeviceTimeout()` should be removed as well.

Reply:
That seemed to have slipped through ... thanks for spotting :+1: